python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -703,11 +622,33 @@ class RandomForestClassifier(Base):
                 raise ValueError('Invalid parameter for estimator')
             else:
                 setattr(self, key, value)
-
+        self.__init__()
         return self
 
     def print_summary(self):
-        self._impl.print_summary()
+
+        cdef RandomForestMetaData[float, int] *rf_forest = \
+            <RandomForestMetaData[float, int]*><size_t> self.rf_forest
+
+        cdef RandomForestMetaData[double, int] *rf_forest64 = \
+            <RandomForestMetaData[double, int]*><size_t> self.rf_forest64
+
+        # Note: self.dtype is initialized in fit.
+        if self.dtype == np.float64:
+            print_rf_summary(rf_forest64)
+        else:
+            print_rf_summary(rf_forest)
 
     def print_detailed(self):
```

Comment:
if this is a public class, it needs a docstring, otherwise chante it to be `_print_detailed(self):`